### ISSUE TITLE
Unable to launch UI on raspberry Pi

### ISSUE THREAD

**Issue by ACWAKKERMANS** [SeydX/homebridge-camera-ui#148]
Hi there,

I am unable to open 192.168.1.177:8181 (Can't connect to server)
In the HomeBridge logs it says:
`[6/2/2021, 7:28:07 AM] [CameraUI] [Interface] An error occured during starting server!
[6/2/2021, 7:28:07 AM] [CameraUI] [Error: EACCES: permission denied, mkdir '/Users'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'mkdir',
  path: '/Users'
}`

How can I fix this?

**Comment by SeydX** [comment]
mkdir /Users/recordings

**Comment by ACWAKKERMANS** [comment]
Thank you for the support!
Is this directory created in the root folder, or in the HomeBridge directory?
I created a Users/recordings directory as advised, but received the same error on restarting HomeBridge

**Comment by SeydX** [comment]
Where u created the directory? As default, camera.ui doesnt create any folder in /Users (root directory)

it seems you have changed the recordings directory in the settings to `/Users`

**Comment by ACWAKKERMANS** [comment]
Everything is installed with the defaults. Including HomeBridge.
The HomeBridge directory is: /var/lib/homebridge.
After installing, the only setting I changed in your plug-in is the adding of one camera name for testing (called "cam") and the RSTP url for that camera.

**Comment by SeydX** [comment]
Can u post your config pls

**Comment by ACWAKKERMANS** [comment]
```
        {
            "name": "CameraUI",
            "port": 8181,
            "theme": "auto",
            "language": "en",
            "debug": false,
            "mqtt": {
                "active": false
            },
            "http": {
                "active": false,
                "localhttp": false
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "Garage Cam",
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-probesize 32 -analyzeduration 0 -rtsp_transport tcp -i rtsp://192.168.1.122/HighResolutionVideo"
                    }
                }
            ],
            "platform": "CameraUI"
        }
```

**Comment by SeydX** [comment]
Can u enable debug (`"debug":true`) pls and post the log again after restart

**Comment by ACWAKKERMANS** [comment]
[6/5/2021, 7:50:16 AM] [Homebridge UI] Changes to config.json saved.
[6/5/2021, 7:50:20 AM] [Homebridge UI] Homebridge restart request received
[6/5/2021, 7:50:20 AM] [Homebridge UI] UI / Bridge settings have not changed; only restarting Homebridge process
[6/5/2021, 7:50:20 AM] [Homebridge UI] Sending SIGTERM to Homebridge
[6/5/2021, 7:50:20 AM] Got SIGTERM, shutting down Homebridge...
[6/5/2021, 7:50:25 AM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[6/5/2021, 7:50:30 AM] [HB Supervisor] Restarting Homebridge...
[6/5/2021, 7:50:30 AM] [HB Supervisor] Starting Homebridge with extra flags: -I -K
[6/5/2021, 7:50:30 AM] [HB Supervisor] Started Homebridge v1.3.4 with PID: 19178

...

[6/5/2021, 7:50:36 AM] [CameraUI] [Plugin] Garage Cam: Configuring unbridged accessory...
[6/5/2021, 7:50:36 AM] [CameraUI] [Plugin] Garage Cam: Setting up accessory...
Preparing Advertiser for 'Garage Cam 3F72' using ciao backend!

...

[6/5/2021, 7:50:36 AM] [CameraUI] [Interface] An error occured during starting server!
[6/5/2021, 7:50:36 AM] [CameraUI] [Error: EACCES: permission denied, mkdir '/Users'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'mkdir',
  path: '/Users'
}
[6/5/2021, 7:50:36 AM] [CameraUI] [Interface] Stopping user interface server...

**Comment by SeydX** [comment]
Is there any folder called „db“ or „recordings“ in `/var/lib/homebridge` ?

Can u also try following
- `mkdir /Users/db`
- `mkdir /Users/recordings` (if removed)

**Comment by ACWAKKERMANS** [comment]
the HomeBridge directory `/var/lib/homebridge` I posted earlier was gotten from the HomeBridge UI. I just browsed there and found no homebridge directory in /var/lib/
I then went to /home and found two folders there: pi and homebridge.
Browsing to /home/homebridge, I found no db or recordings folders there. Would you like me to create them?

I did follow part two of your suggestions and created a db directory in /Users

**Comment by SeydX** [comment]
Where is your config.json located exactly?

camera.ui wants to create the db/recordings folder in `/Users` because it thinks there is your homebridge located for some reason

**Comment by ACWAKKERMANS** [comment]
OK...
For some reason, via SSH I did not see the directory `/var/lib/homebridge` as I said before.
I decided to log into the RPI UI and was able to find the directory there. 
I also found that the Users folder I thought I had created in the root was actually under "Home".
I went into the root and created `/Users/db and /Users/recordings`.
After a restart, I had the same error message, but now it wanted a `/Users/Home 1/` directory
I created this and after a restart it wanted a `/Users/Home 1/.homebridge/` directory
After another restart it asked for a` /Users/Home 1/.homebridge/recordings/` directory
After creating that and restarting again... it finally worked!
I honestly don't understand how all this came to be...

Thanks very much for your support! I'll have a go at the camera UI and see if any further related issues come up
